systemtest/src/test/java/io/strimzi/systemtest/mirrormaker/MirrorMakerST.java

```diff
@@ -453,12 +453,6 @@ void testMirrorMakerTlsScramSha(ExtensionContext extensionContext) {
             .build();
 
         internalKafkaClient.produceAndConsumesTlsMessagesUntilBothOperationsAreSuccessful();
-
-        InternalKafkaClient newInternalKafkaClient = internalKafkaClient.toBuilder()
-            .withClusterName(kafkaClusterTargetName)
-            .withKafkaUsername(userTarget.getMetadata().getName())
-            .build();
-
         internalKafkaClient.consumesTlsMessagesUntilOperationIsSuccessful(internalKafkaClient.getMessageCount());
```

Comment:
Maybe the `newInternalKafkaClient` should be used in the `internalKafkaClient.consumesTlsMessagesUntilOperationIsSuccessful(internalKafkaClient.getMessageCount());`?

Reply:
That might be possible. Checking..